api/swagger.yaml

```diff
@@ -1398,44 +1398,51 @@ definitions:
     type: "object"
     properties:
       Bridge:
-        description: Name of the network's bridge (for example, `docker0`).
+        description: |
+          Name of the default bridge interface when dockerd's --bridge flag is set.
         type: "string"
-        example: "docker0"
+        example: ""
```

Comment:
Is this deprecated too? Otherwise probably no need to remove this example.

Reply:
Ah! Good one; it looks like another commit in that PR added it back; https://github.com/moby/moby/pull/45945/commits/841932caa77e583e0d3812b2c573b23bb2995135

Let me merge those changes to the swagger file into this commit.

Reply:
updated; PTAL

Reply:
Oh, looks like I amended the wrong commit :see_no_evil: